addons/website_crm_partner_assign/static/src/scss/crm_partner_assign.scss

```diff
@@ -0,0 +1,39 @@
+.short_description {
+    max-height: 100px;
+    display: -webkit-box;
+    -webkit-line-clamp: 4;
+    -webkit-box-orient: vertical;
+}
+
+.partner_details_contact address > div div {
+    padding: map-get($spacers, 1) 0;
+
+    &:last-child{
+        border: none;
+    }
+}
+
+.filters_offcanvas {
```

Comment:
Always prefix custom classes with `o_`.
Since the component is intended to be used for this module only, you can extend the prefix with the module abbr.
Eg. in `website_sale` we use `o_wsale_`.

If the module doesn't have a convention for class names you could establish one.